### PR TITLE
Update formal-review-process.md

### PR DESCRIPTION
**Pull request by yomiadetutu1**
Updated page to reflect Lattice

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 1]
```diff
@@ -12,23 +12,23 @@ Before each review cycle, Managers read the performance review chapter in High O
...
+All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-6 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself).
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md]:
```suggestion
All full-time staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-6 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself). 
```

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 2]
```diff
@@ -12,23 +12,23 @@ Before each review cycle, Managers read the performance review chapter in High O
...
...il the end of the feedback stage. More information for managers on managing the peer selection process is available [here](https://cloud.scorm.com/ScormEngineInterface/defaultui/player/modern.html?configuration=db1M4iPRqeQcV5M7WEvXeR548xreqOrdBfArCusspmFNak_ADJYNngS4sDh3tatitixq2pK56M4ismKY6pGxKFiSJwoc_RMzaLmfg4Orl-taMYTOGgoiqN7_BQOf_4gsnGXazYQNTvQQCm-TJaCY1GLMORukVcukS7mKtH2eDi4DZ8BLS2d_bz5cIK0FGjaasCG6eRal8CY89VULtdgkzMAeMDLfHK0FSpUAik6ssQWzH2o4kJXtEAD8DUyXIE2vMoFVtcWQiQSD-RQ2YQS4SUPct2_cAALcy_lID7dplGN4A5SS_VooWwYKmbW_ImAVf0d1B_yIn_Wh6bIBQwR14UIs8nRIfJp4RYjp44Mvwt6V91K-r3vnYUl0glsburdcjT_LDskdNVXdQ6mUKiPhYQ&preventRightClick=false&cc=en_US&cache=21.1.29.480&playerConfUrl=j&package=4FUGRO2eEOyBQAZP2finOVivEu2TdjeQ6cDnrisganACU6vl5LGF6TqYZ_w9VxSkYld_2FKIQQc_eHyCjhxgA1JaEyUw3rYSzv1r4Qwz_stAvAkbDhJUTA&redirectUrl=https%3A%2F%2Flatticeuniversity.lattice.com%2Fcollections%2Fbf3046c4-d8b0-11ed-a471-06c0361096e5%3Frefresh_status%3Dtrue&registration=URU6tIU2GtP0JEIop33AIAPQ7P9NFOOYbvmvbI8gVfRjLPf7MrV7m7QpMzaDrRbTE_y_Ins1o7Te6K_jBkBohg0DCuNsz6LUxVSs_WcMuJcgXIPlo2sQT8jddmwMOisDwLsZRkhYxvE0HUWLzW4Ii6qRS5Xn5DtM-bTuQI0PNVROx6jz&tracking=true&forceReview=false&player=modern&ts=20231106234140).
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:17]:
```suggestion
Once your direct report has completed their nominations, Managers will receive an email and a task in your Lattice home page inbox. You are able to either (a) approve your direct report's nominations and immediately send feedback requests to their peer selection, or (b) edit their nominations or add new peers for additional feedback. You can find more information on how to approve your direct reports' peer nominations [here](https://help.lattice.com/hc/en-us/articles/1500001692742-Approve-Your-Direct-Report-s-Peer-Selection). Depending on the role, part-time staff and community members may be solicited to provide feedback in addition to the peer panel selected in Lattice. Once complete, your requests will be scheduled to be sent when the feedback stage starts. Managers may return to this page to send more requests up until the end of the feedback stage. More information for managers on managing the peer selection process is available [here](https://cloud.scorm.com/ScormEngineInterface/defaultui/player/modern.html?configuration=db1M4iPRqeQcV5M7WEvXeR548xreqOrdBfArCusspmFNak_ADJYNngS4sDh3tatitixq2pK56M4ismKY6pGxKFiSJwoc_RMzaLmfg4Orl-taMYTOGgoiqN7_BQOf_4gsnGXazYQNTvQQCm-TJaCY1GLMORukVcukS7mKtH2eDi4DZ8BLS2d_bz5cIK0FGjaasCG6eRal8CY89VULtdgkzMAeMDLfHK0FSpUAik6ssQWzH2o4kJXtEAD8DUyXIE2vMoFVtcWQiQSD-RQ2YQS4SUPct2_cAALcy_lID7dplGN4A5SS_VooWwYKmbW_ImAVf0d1B_yIn_Wh6bIBQwR14UIs8nRIfJp4RYjp44Mvwt6V91K-r3vnYUl0glsburdcjT_LDskdNVXdQ6mUKiPhYQ&preventRightClick=false&cc=en_US&cache=21.1.29.480&playerConfUrl=j&package=4FUGRO2eEOyBQAZP2finOVivEu2TdjeQ6cDnrisganACU6vl5LGF6TqYZ_w9VxSkYld_2FKIQQc_eHyCjhxgA1JaEyUw3rYSzv1r4Qwz_stAvAkbDhJUTA&redirectUrl=https%3A%2F%2Flatticeuniversity.lattice.com%2Fcollections%2Fbf3046c4-d8b0-11ed-a471-06c0361096e5%3Frefresh_status%3Dtrue&registration=URU6tIU2GtP0JEIop33AIAPQ7P9NFOOYbvmvbI8gVfRjLPf7MrV7m7QpMzaDrRbTE_y_Ins1o7Te6K_jBkBohg0DCuNsz6LUxVSs_WcMuJcgXIPlo2sQT8jddmwMOisDwLsZRkhYxvE0HUWLzW4Ii6qRS5Xn5DtM-bTuQI0PNVROx6jz&tracking=true&forceReview=false&player=modern&ts=20231106234140).
```

**Review by cwarnermm — DISMISSED** [review]
Feedback provided inline. Approving.

**Review by lynn915 — CHANGES_REQUESTED** [review]
@yomiadetutu1 can you update the recommended number of reviewers to be 3-5? 

Also, for this part, can you update it to: 
Staff who have received peer feedback requests will receive an email and a task on their Lattice home page. To complete the request, click **give feedback**. You may decline the request if you don't have feedback to share; however, staff are encouraged to provide feedback when selected as a peer reviewer. Peer reviews help give managers a fuller picture by removing potential blindspots on how their direct reports are performing as it relates to their AORs, goals and growth opportunities, which ultimately helps your peers improve performance and growth potential at Mattermost.

File: operations/workplace/people/performance-reviews-50/formal-review-process.md (hunk 1 above)
Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md]:
```suggestion
All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-5 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself). 
```

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 3]
```diff
@@ -12,23 +12,23 @@ Before each review cycle, Managers read the performance review chapter in High O
...
+Prior to selecting peer panels and writing performance reviews for their direct reports, managers should review Lattice's guide on navigating reviews for managers [here](https://help.lattice.com/hc/en-us/articles/1500002045142-Navigate-Reviews-as-a-Manager).
 
 ## Peer feedback
 
-Staff who have received peer feedback requests will receive an email and a task in their Culture Amp inbox. To complete the request, click **give feedback**. Peers may decline the request based on bandwidth (more than 5 peer feedback requests), if they have give feedback in the previous review cycle and feel that their feedback would be duplicative, or if they don't have feedback to share. 
+Staff who have received peer feedback requests will receive an email and a task on their Lattice home page. To complete the request, click **give feedback**. Peers may decline the request based on bandwidth (more than 5 peer feedback requests), if they have give feedback in the previous review cycle and feel that their feedback would be duplicative, or if they don't have feedback to share.
```
Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md]:
```suggestion
Staff who have received peer feedback requests will receive an email and a task on their Lattice home page. To complete the request, click **give feedback**. You may decline the request if you don't have feedback to share; however, staff are encouraged to provide feedback when selected as a peer reviewer. Peer reviews help give managers a fuller picture by removing potential blindspots on how their direct reports are performing as it relates to their AORs, goals and growth opportunities, which ultimately helps your peers improve performance and growth potential at Mattermost.  
```

**Review by lynn915 — CHANGES_REQUESTED** [review]
@yomiadetutu1 - can you please update this to 3-5 (not 6)
"We recommend selecting 3-6 peers who can effectively provide feedback on achievements and performance over the past 6 months."

For comp review, can you update to approval by CFO and CEO (there is no longer a VP Finance).

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ Managers will share their review expectation ratings \(meets, exceeds, does not
 
 ## Compensation review and change memos
 
-All compensation changes require approval of the VP of Finance & Operations and the CEO. CTO approval is required ahead of CEO approval for all Engineering compensation changes.
+All compensation changes require approval of the VP of Finance and the CEO.
```
Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md]:
```suggestion
All compensation changes require approval of the CFO and the CEO.
```

File: operations/workplace/people/performance-reviews-50/formal-review-process.md [hunk 5]
```diff
@@ -12,23 +12,23 @@ Before each review cycle, Managers read the performance review chapter in High O
...
+All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-5 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself).
```
Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:15]:
```suggestion
All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-5 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself). 
```

Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:15]:
```suggestion
All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-5 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself). 
```

Comment (yomiadetutu1) [inline operations/workplace/people/performance-reviews-50/formal-review-process.md:15]:
```suggestion
All FT staff participating in the review cycle will receive a notification via email as well as a task on your Lattice home page asking you to nominate peers for feedback. To get started, click **choose peer reviewers** from the email notification -OR- click the **Select your peer reviewers** task from your Lattice Home page and search for the peers you'd like to nominate. (Note that Lattice pulls its information directly from BambooHR, so legal names will appear in Lattice. If you can't find a peer, please reach out to hr@mattermost.com). On the nomination page, enter the names of the peers you would like to nominate. Review and Confirm nominations by clicking **submit nominations**. We recommend selecting 3-5 peers who can effectively provide feedback on achievements and performance over the past 6 months. Peer nomination selections will be sent to your manager to be finalized. More information on peer nominations is available [here](https://help.lattice.com/hc/en-us/articles/1500001692902-Nominate-Peer-Reviewers-for-Yourself). 
```

**Review by lynn915 — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**